### ISSUE TITLE
Display first execution slowness reason

### ISSUE THREAD

**Issue by pgreze** [tldr-pages/tldr-c-client#34]
## Reproduction Rate
Always?

## Steps to Reproduce
When using tldr for the first time.

## Result
First start is REALLY slow. After it works like a charm.

## Expected Result
I guess initial data are downloaded/unzipped/whatever
but I was thinking I was using the (slow?) Python client.

I realized after than homebrew was already installing this client by default and consecutive executions were really fast.

So why not display an initial message explaining that we're already using an optimized client and that only first start is slow?

## Additional Information
```
➜  ~ $ tldr --version
tldr v1.3.0 (v1.3.0)
Copyright (C) 2016 Arvid Gerstmann
Source available at https://github.com/tldr-pages/tldr-cpp-client
```
